### PR TITLE
Improved instructions for cluster installation

### PR DESCRIPTION
**Pull request by mbudiu-vmw**

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
@daniarherikurniawan : hopefully these instructions are enough to install on a cluster

File: README.md [hunk 1]
```diff
@@ -179,6 +179,14 @@ machine as a one-machine cluster.
 
 ## 3.2 Deployment scripts
 
+First install Hillview locally:
+
+```
+$ bin/install.sh
+```
+
+Edit the config.json file as described above.
```
Comment (daniarherikurniawan) [inline README.md:188]: After this, the next instruction is `deploy.py`, but there is no `target/` directories to deploy. I believe there should be 1 more step which is 'bin/rebuild.sh'.  When you wrote `First install Hillview locally:`, it means that we should clone this master repo, right?

Comment (mbudiu-vmw) [reply README.md:188]: If you didn't clone the repository you don't have this file either.

Comment (daniarherikurniawan) [reply README.md:188]: Got it! In my case, I still need to run `bin/rebuild.sh` before deploying it

Comment (mbudiu-vmw) [reply README.md:188]: what happens if you don't?

**Pull request merged**